docs/Policy_definition.rst

```diff
@@ -91,8 +91,32 @@ App-AutoScaler Policy Definition
 | initial_min_instance_count           | int                        | false   | the initial minimal number of instance count for this schedule             |
 +--------------------------------------+----------------------------+---------+----------------------------------------------------------------------------+
 
+**Note**
 
-**Reference**
+You need to aware the following facts when define multiple schedules in policy:
+
+* If one schedule overlaps another, the one which starts first will be guaranteed, while the later one is completely ignored. For example: 
+
++-----+---------------------------------------------------------------------------------------+
+|Index|Schedule                                                                               |
++-----+---------------------------------------------------------------------------------------+
+|1    |8:00AM ~ 8:00PM every Tuesday , with instance_min_count=1, instance_max_count=10       |
```

Comment:
give an example of three schedules, 
```
schedule 1:      |-----------| 
schedule 2:             |----------------------|
schedule 3:                               |-------|
```
In this scenario,  schedule 1, 3 will be applied, schedule 2 will be ignored.